http/src/main/scala/filodb/http/PrometheusApiRoute.scala

```diff
@@ -43,7 +45,7 @@ class PrometheusApiRoute(nodeCoord: ActorRef, settings: HttpSettings)(implicit a
           val logicalPlan = Parser.queryRangeToLogicalPlan(query, TimeStepParams(start.toLong, step, end.toLong))
 
           askQueryAndRespond(dataset, logicalPlan, explainOnly.getOrElse(false), verbose.getOrElse(false),
-            spread, PromQlQueryParams(query, start.toLong, step.toLong, end.toLong, spread))
+            spread, PromQlQueryParams(ConfigFactory.empty, query, start.toLong, step.toLong, end.toLong, spread))
```

Comment:
Can you document why we pass `ConfigFactory.empty` : `No cross-cluster failure routing in this API, hence we pass empty config`